modules/ecs6-class/point.js

```diff
@@ -1,12 +1,21 @@
 class Point {
-    constructor({x=0, y=0}={}) {
+    constructor({ x = 0, y = 0 } = {}) {
+        if (typeof (x) !== 'number' || typeof (y) !== 'number')
+            throw Error('x or y are not a number')
```

Comment:
I need to know exactly what the error is

Reply:
I changed what needed to be changed.